features/admin.users.v1/components/user-profile.tsx

```diff
@@ -2000,6 +2064,34 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
                                                                     </div>
                                                                 )
                                                     }
+                                                    {
+                                                        showPendingVerificationPopup(value)
+                                                            && (
+                                                                <div
+                                                                    className="verified-icon"
+                                                                    data-componentid={
+                                                                        `${testId}-profile-form-${schema.name}` +
+                                                                        `-pending-verification-icon-${index}`
+                                                                    }
+                                                                >
```

Comment:
Do we need this `<div/>` element?

Reply:
The Popup component doesn't trigger correctly when directly passed to Semantic UI's <Field> icon prop, as Semantic UI internally modifies the icon element, interfering with Popup's event binding. Wrapping Popup within a stable HTML element (e.g., <div> or <span>) ensures Popup attaches hover events properly.